packages/dev/parcel-transformer-docs/DocsTransformer.js

```diff
@@ -508,6 +508,10 @@ module.exports = new Transformer({
         });
       }
 
+      if (path.isTSModuleDeclaration()) {
```

Comment:
honestly, not sure what to do with the shape of this, is just returning node good enough?

Reply:
why was it added?

Reply:
because https://github.com/adobe/react-spectrum/blob/2e510ba1bb22f49ecd41506eadcf13a68a43c267/packages/dev/parcel-transformer-docs/DocsTransformer.js#L248 will return undefined when an unrecognized type comes through, then we check properties on it that returned value on the next line, causing it to blow up
so options are, to recognize the type, or to do an undefined check and try filling in the value we're going to return for a qualified name based on an unrecognized type